### PR TITLE
SpotRequest data retrieval

### PR DESCRIPTION
**Pull request by stephanlindauer**
@rebuy-de/prp-cost-exporter please review

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -21,11 +21,11 @@ func (app *App) Run(cmd *cobra.Command, args []string) {
 
 	config := config.Parse(app.config)
 
-	coreRetriever := retriever.CoreRetriever{
+	ApiRetriever := retriever.ApiRetriever{
```
Comment (svenwltr) [inline cmd/root.go]: Correct casing would be `APIRetriever`

File: pkg/prom/model.go [hunk 2]
```diff
@@ -38,6 +39,15 @@ var C = Costs{
 		},
 		[]string{"account"},
 	),
+	SpotRequest: *prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "rebuy",
+			Subsystem: "cost_exporter",
```
Comment (svenwltr) [inline pkg/prom/model.go]: We might have to change this?

File: pkg/retriever/api.go [hunk 3]
```diff
@@ -80,3 +83,31 @@ func (c *CoreRetriever) getCores() {
...
+				"region":            service.Region,
+				"state":             *request.State,
+				"code":              *request.Status.Code,
+				"instance_type":     *request.LaunchSpecification.InstanceType,
+				"instance_id":       *request.InstanceId,
+				"availability_zone": *request.LaunchedAvailabilityZone,
+			}
+			spotRequestItems = append(spotRequestItems, prom.SpotRequestItem{
+				Labels: label,
+				Count:  1,
```
Comment (svenwltr) [inline pkg/retriever/api.go]: You actually do not need this `Count`, right? So `spotRequestItems` could actually be of the type `[]prometheus.Labels`.

File: pkg/prom/model.go (hunk 2 above)
Comment (stephanlindauer) [reply pkg/prom/model.go]: that would be a bigger thing, i guess, with renaming the repo and so on. 
i would do that next friday.

File: pkg/retriever/api.go (hunk 3 above)
Comment (stephanlindauer) [reply pkg/retriever/api.go]: yea i guess i was saving that for some future purpose that will never come up probably. ;)

File: cmd/root.go (hunk 1 above)
Comment (stephanlindauer) [reply cmd/root.go]: true. fixed.

**Pull request merged**